introduction/07-faq.asciidoc

```diff
@@ -0,0 +1,15 @@
+== FAQ
+
+To conclude the Introduction segment of the Learning Path, here are some Frequently Asked Questions people have when embarking on their InnerSource journey.
+
+=== What is the cost/overhead?
+
+=== Why not just open source?
+
+=== Will this slow us down?
+
+=== Will we just be reviewing PRs all day every day?
+
+=== How do we convince management this is a good idea?
+
+=== How do we convince engineers this is a good idea?
```

Comment:
Make it fun.

Reply:
I've got some specific concerns that I need addressing:
- Losing control of your project
- Being judged for the quality of your code
- The possibility of rejecting PRs requires psychological safety (on both sides)

Reply:
> Losing control of your project

Through the `Trusted Committer` and `Product Owner` roles you can still ensure that the incoming code is a good fit from both a product and engineering perspective.  You do not have to merge code that is not a good fit.

> Being judged for the quality of your code

This goes with the psychological safety point below.
A growth mindset welcomes feedback.
A fixed mindset will have trouble with this.

> The possibility of rejecting PRs requires psychological safety (on both sides)

This is a change in mindset from rejecting a PR vs. partnering to work on it together so it can be accepted.

Reply:
Ah yes I missed this one! Included the points raised here in https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/465/commits/0f1e46c65aae62a67e74d850d37ff732b8ff7605.